Makefile

```diff
@@ -23,8 +23,8 @@ DATE = $(shell date +%F_%H-%M-%S)
 # | suse             | sles12sp5, sle15              |                                                                |
 # | freebsd          |                               | Not supported                                                  |
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-OS_RELEASE  ?= debian
-OS_VERSION  ?= bullseye-slim
+OS_RELEASE  ?= ubuntu
+OS_VERSION  ?= 22.04
```

Comment:
Is this change intentional? I would assume this was just for local testing and should be reverted before merging?